example/sqs.tf

```diff
@@ -1,12 +1,15 @@
 module "example_sqs" {
-  source = "github.com/ministryofjustice/cloud-platform-terraform-sqs?ref=3.3"
+  source = "github.com/ministryofjustice/cloud-platform-terraform-sqs?ref=3.5"
```

Comment:
Why 3.3 -> 3.5, not 3.4?

Reply:
We already have a 3.4 that is the latest, but some one forgot to update the example here